src/django/api/matching.py

```diff
@@ -90,6 +90,18 @@ def get_canonical_items():
     return items
 
 
+def sort_exact_matches(exact_matches, active_item_ids, contributor):
+    def sort_order(f):
+        is_active = f['facility_id'] in active_item_ids
```

Comment:
This comparison looks like it may be incorrect. If I am reading the code property, `active_item_ids` is a list of `FacilityListItem` ids. A `Facility` id read from `f['facility_id']` will never be ina list of `FacilityListItem` ids.